linode/manager
# M3-4478 Add: access management for OBJ access keys

## Description

Add limited access controls for OBJ access keys. See the JIRA ticket for details.

## Note to Reviewers

Work for this is done, but some changes to the API for this endpoint are in the works, so marking it `wait` until that's ironed out.

To test:

Create access keys of both varieties (with and without the "Limited Access" toggle active). Check the payload and API response, if limited is not toggled, `bucket_access: null` should be sent, and that should come back from the API along with `limited: false`. If the switch is toggled, the selected permissions should be included in request and response payloads.

If you try to create a key with access to 0 buckets (select "limited access" and set all the radio buttons to "none"), you'll end up with a non-limited key--this is the bit the API has to fix.
